TerrariaServerAPI/TerrariaApi.Server/HookManager.cs

```diff
@@ -55,6 +55,13 @@ public void AttachOTAPIHooks(string[] args)
 			Hooking.ServerHooks.AttachTo(this);
 			Hooking.WiringHooks.AttachTo(this);
 			Hooking.WorldHooks.AttachTo(this);
+
+			//Run after Hooking.ServerHooks.AttachTo(this) to override the Hooks.Command.StartCommandThread hook
+			if (Console.IsInputRedirected && args.Any(x => x == "-redirect"))
+			{
+				ServerApi.LogWriter.ServerWriteLine($"TerrariaServer is running in the background. Input has been re-enabled.", TraceLevel.Info);
```

Comment:
This message is just a bit unclear, at least to me.

```suggestion
				ServerApi.LogWriter.ServerWriteLine($"TerrariaServer is running in the background, but input was re-enabled by command line flag.", TraceLevel.Info);
```